modules/social_features/social_like/social_like.module

```diff
@@ -18,5 +20,50 @@ function social_like_entity_view_alter(array &$build, EntityInterface $entity, E
     if (!\Drupal::currentUser()->hasPermission('view like widget')) {
       unset($build['like_and_dislike']);
     }
+    else {
+      $build['#attached']['library'][] = 'core/drupal.dialog.ajax';
+    }
   }
 }
+
+
+/**
+ * Implements hook_views_pre_render().
+ *
+ * @param \Drupal\views\ViewExecutable $view
```

Comment:
Hooks should not have @param in their function doc comment, only a description if needed.